client/components/Search.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { setSearchInput } from '../reducers'
+
+const Search = (props) => {
+  return (
+    <form name="search">
+      <input
+        onChange={props.handleChange}
+        value={props.searchInput}
+        placeholder="Search by name"
+        name="searchInput"
+      />
+    </form>
+  )
+}
+
+const mapStateToProps = state => {
+  return {
+    searchInput: state.searchInput
+  }
+}
+
+const mapDispatchToProps = dispatch => {
+  return {
+    handleChange: (evt) => {
+      dispatch(setSearchInput(evt.target.value))
+    }
+  }
+}
+
```

Comment:
I think it would be easier to embed the Search form in the navbar, and whenever a search input is "submitted" we will redirect to `/` with the filtered categories. I think it would be too clunky to have a separate `/search`  i.e. `/` --> click on Search Button --> Display `/search` page --> input search -->submit search --> re-direct and display filtered categories at `/`. 

INSTEAD:  `Search` component is embedded in the navbar
When on homepage: all products filter by category, no redirect just update display 
When on any other page that is not `/`: redirect back to home with filtered display

Reply:
jamie moved this to navbar and figured out how to change the font input color to black